releasenotes/notes/circuit-assign-parameter-to-concrete-value-7cad75c97183257f.yaml

```
@@ -0,0 +1,13 @@
+---
+fixes:
+  - |
+    Changed :meth:`~qiskit.circuit.QuantumCircuit.assign_parameters` to bind
```

```suggestion
    Changed :meth:`.QuantumCircuit.assign_parameters` to bind
```

super minor, but this will just cause the cross-reference text to display `QuantumCircuit.assign_parameters` rather than the previous form which would only show `assign_parameters` without the context.